report.ipynb

```diff
@@ -0,0 +1,678 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Naravni zlepek\n",
+    "\n",
+    "Danih je $n$ interpolacijskih točk $(x_i, y_i), i=1,2,...,n$. Naravni interpolacijski zlepek $S$ je funkcija, ki izpolnjuje naslednje pogoje:\n",
+    "\n",
+    "1. $S(x) = y_i, \\quad i=1,2,...,n$\n",
+    "2. $S$ je polinom stopnje **3 ali manj** na vsakem podintervalu $[x_i, x_{i+1}], i=1,2,...,n-1$\n",
+    "3. $S$ je dvakrat zvezno odvedljiva funkcija na interpolacijskem intervalu $[x_1, x_n]$\n",
+    "4. $S''(x_1) = S''(x_n) = 0$\n",
+    "\n",
+    "Zlepek $S$ določimo tako, da predpostavimo:\n",
+    "$$\n",
+    "S(x) = S_i(x) = a_i + b_i(x-x_i) + c_i(x-x_i)^2 + d_i(x-x_i)^3, \\quad x \\in [x_i, x_{i+1}]\n",
+    "$$\n",
+    "nato pa izpolnemo zahtevane pogoje."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Vemo, da $S''(x_1) = S''(x_n) = 0$, torej lahko formuliramo drugi odvod zlepka, ki deluje na vrednostih $(x_i, z_i)$ kot:\n",
+    "$$\n",
+    "S''_i(x) = z_i \\frac{x - x_{i+1}}{x_i - x_{i+1}} + z_{i+1} \\frac{x - x_i}{x_{i+1} - x_i}\n",
```

Comment:
Kako pridemo do te enačbe? Izpeljavo bodisi napišite v celoti ali pa jo izpustite in navedite le sistem enačb in povezavo rešitve s koeficienti $a_i, b_i, c_i$ in $d_i$.